oio/crawler/integrity.py

```diff
@@ -791,9 +825,11 @@ def main():
         request_attempts=args.attempts,
         verbose=True,
     )
-    entries = csv.reader(source, delimiter=' ')
-    for entry in entries:
-        checker.check(Target(*entry))
+    if entries:
+        for entry in entries:
+            checker.check(Target(*entry), recurse=DEFAULT_DEPTH)
```

Comment:
Since the Target constructor has been changed (oio/crawler/integrity.py L45), we should add more tests using CSV input

Reply:
The `Target` constructor argument order has been restored to what it was initially. I don't know if the standard input feature is actually used, but yes, we could add some tests.